fmn/rules/utils.py

```diff
@@ -198,6 +244,58 @@ def _get_pkgdb2_packages_for(config, username, flags):
     return dict(packages)
 
 
+def _get_pagure_packages_for(config, username, flags):
+    """
+    Get the packages a user is associated with from pagure.
+
+    Args:
+        config (dict): The application configuration.
+        username (str): The FAS username to fetch the packages for.
+        flags (list): The type of relationship the user should have to the
+            package (e.g. "watch", "point of contact", etc.).
+    """
+    log.debug("Requesting pagure packages for user %r" % username)
+
+    # XXX - We don't yet have a way to query for all projects that a user
```

Comment:
Could you file this as an issue that points to this and remove the TODO from the code?